site/content/en/docs/Guides/troubleshooting.md

```diff
@@ -5,6 +5,206 @@ weight: 200
+    Image:          gcr.io/agones-mark/agones-sdk:1.0.0-dce1546
+    Image ID:       docker-pullable://gcr.io/agones-mark/agones-sdk@sha256:4b5693e95ee3023a2b2e2099d102bb6bac58d4ce0ac472e58a09cee6d160cd19
+    Port:           <none>
+    Host Port:      <none>
+    State:          Running
+      Started:      Thu, 15 Aug 2019 22:12:31 +0000
+    Ready:          True
+    Restart Count:  0
+    Requests:
+      cpu:     30m
+    Liveness:  http-get http://:8080/healthz delay=3s timeout=1s period=3s #success=1 #failure=3
+    Environment:
+      GAMESERVER_NAME:  xonotic-r58tb-5rzr4
+      POD_NAMESPACE:    default (v1:metadata.namespace)
+    Mounts:
+      /var/run/secrets/kubernetes.io/serviceaccount from agones-sdk-token-vr6qq (ro)
+Conditions:
+  Type              Status
+  Initialized       True
+  Ready             True
+  ContainersReady   True
+  PodScheduled      True
+Volumes:
+  empty:
+    Type:    EmptyDir (a temporary directory that shares a pod's lifetime)
+    Medium:
+  agones-sdk-token-vr6qq:
+    Type:        Secret (a volume populated by a Secret)
+    SecretName:  agones-sdk-token-vr6qq
+    Optional:    false
+QoS Class:       Burstable
+Node-Selectors:  <none>
+Tolerations:     node.kubernetes.io/not-ready:NoExecute for 300s
+                 node.kubernetes.io/unreachable:NoExecute for 300s
+Events:
+  Type    Reason     Age    From                                             Message
+  ----    ------     ----   ----                                             -------
+  Normal  Scheduled  9m3s   default-scheduler                                Successfully assigned default/xonotic-r58tb-5rzr4 to gke-test-cluster-default-590db5e4-jgc5
+  Normal  Pulling    9m2s   kubelet, gke-test-cluster-default-590db5e4-jgc5  pulling image "gcr.io/agones-images/xonotic-example:0.6"
+  Normal  Pulled     7m59s  kubelet, gke-test-cluster-default-590db5e4-jgc5  Successfully pulled image "gcr.io/agones-images/xonotic-example:0.6"
+  Normal  Created    7m56s  kubelet, gke-test-cluster-default-590db5e4-jgc5  Created container
+  Normal  Started    7m56s  kubelet, gke-test-cluster-default-590db5e4-jgc5  Started container
+  Normal  Pulling    7m56s  kubelet, gke-test-cluster-default-590db5e4-jgc5  pulling image "gcr.io/agones-mark/agones-sdk:1.0.0-dce1546"
+  Normal  Pulled     7m54s  kubelet, gke-test-cluster-default-590db5e4-jgc5  Successfully pulled image "gcr.io/agones-mark/agones-sdk:1.0.0-dce1546"
+  Normal  Created    7m54s  kubelet, gke-test-cluster-default-590db5e4-jgc5  Created container
+  Normal  Started    7m54s  kubelet, gke-test-cluster-default-590db5e4-jgc5  Started container
+```
+
+Finally, you can also get the logs of your `GameServer` `Pod` as well via `kubectl logs <pod name> -c <game server container name>`, or use
+your Kubernetes platform of choice's logging aggregation tools. Check [How do I see the logs for Agones?](#how-do-i-see-the-logs-for-agones)
+for more details on retrieving logs.
```

Comment:
This link goes to the section about getting system logs, which is a bit of a disconnect from getting logs from gameservers. Maybe start this sentence with "If you want to see logs for Agones to see how the system is behaving, check ..."

Reply:
Removed section pointing to below, you are right, it was confusing.